src/app/components/speeddial/speeddial.ts

```diff
@@ -86,7 +86,7 @@ import { asapScheduler } from 'rxjs';
                     [attr.data-pc-section]="'menuitem'"
                 >
                     <a
-                        *ngIf="isClickableRouterLink(item); else elseBlock"
+                        *ngIf="_visible && isClickableRouterLink(item); else elseBlock"
```

Comment:
Hi, mb use only one `ngIf`?

```html
<ng-container *ngIf="_visible">
  <a *ngIf="isClickableRouterLink(item); else elseBlock"></a>
  <ng-template #elseBlock><a> </a></ng-template>
</ng-container>
```

Reply:
Hey,you right would be a cleaner solution. But as you are saying it, maybe its better to pack it into the isClickableRouterLink method isn't it? Then the hole method would return false if not visible and blocks also other points where the method might be used.